Xamarin.Forms.Platform.iOS/Renderers/UIContainerCell.cs

```diff
@@ -0,0 +1,71 @@
+﻿using UIKit;
+
+namespace Xamarin.Forms.Platform.iOS
+{
+	public class UIContainerCell : UITableViewCell
+	{
+		private IVisualElementRenderer _renderer;
+		private object _bindingContext;
+
+		public UIContainerCell(string cellId, View view) : base(UITableViewCellStyle.Default, cellId)
+		{
```

Comment:
User could user this I wonder if the flag should cover this usage.

Reply:
Drop
```C#
	private
```
From
```C#
       private object _bindingContext;
```